<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gp_gestaocolab</class>
 <widget class="QDialog" name="gp_gestaocolab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>781</width>
     <height>361</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_novocolaborador">
    <attribute name="title">
     <string>Novo Colaborador</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>301</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="btn_gravarNovoColab">
        <property name="text">
         <string>Gravar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_cancelarNovoColab">
        <property name="text">
         <string>Cancelar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>751</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Nome</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_nomeNovoColab"/>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Telefone</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_telefoneNovoColab"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>691</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string> Username:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_usernameNovoColab"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string> Senha:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_senhaNovoColab">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Tipo de Acesso</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cb_acessoNovoColab"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_gestaocolab">
    <attribute name="title">
     <string>Gestao Colaboradores</string>
    </attribute>
    <widget class="QTableWidget" name="tw_listacolab">
     <property name="geometry">
      <rect>
       <x>395</x>
       <y>101</y>
       <width>371</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>351</width>
       <height>311</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_12">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Nome</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_nomeGestaoColab"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Telefone</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_telefoneGestaoColab"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Username</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_usernameGestaoColab"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Senha</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_senhaGestaoColab">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Acesso</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cb_acessoGestaoColab"/>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="btn_gravarGestaoColab">
          <property name="text">
           <string>Gravar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_excluirGestaoColab">
          <property name="text">
           <string>Excluir</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_vervendasGestaoColab">
          <property name="text">
           <string>Ver Vendas</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>10</y>
       <width>218</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_13">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Filtrar por:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_idcolabGestaoColab">
          <property name="text">
           <string>ID Colab.</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_nomeGestaoColab">
          <property name="text">
           <string>Nome</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_15">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_14">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Filtro</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_filtroGestaoColab"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="btn_filtrarGestaoColab">
          <property name="text">
           <string>Pesquisar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
